Hi3Helper.Core/Classes/Shared/Region/LauncherConfig.cs

```diff
@@ -309,6 +315,9 @@ public static bool IsShowRegionChangeWarning
             { "MinimizeToTray", false },
             { "UseExternalBrowser", false },
             { "EnableWaifu2X", false },
+            { "BackgroundAudioVolume", 0.75d },
```

Comment:
Honestly I would set it to `0.50d` so that it's not too loud (you never know), but it works either way